### DISCUSSION TITLE
Domains llamacpp.org and ggml.org

### DISCUSSION THREAD

**Discussion by zhangxiangxiao** [ggml-org/llama.cpp#3104] [General]
Found these available and bought them. For now both domains are redirected to the corresponding github repos. Happy to transfer to any one in charge. Also happy to keep them redirected in this way indefinitely.

**Comment by JohannesGaessler** [comment]
Ever since yesterday I am getting constant unsolicited spam with offers for SEO or webdev for "my website". Did you in any way shape or form attach my email address to the domain? (There's a good chance it was simply scraped.)

**Reply by zhangxiangxiao** [reply]
Nope, the domains are registered under my own email with privacy protection enabled.
